styles/components/_masthead.scss

```diff
@@ -317,23 +317,85 @@
 // Selector Dropdown
 // ==================================================================
 .sprk-c-Masthead__selector-dropdown {
+  background-color: $sprk-white;
   border-bottom: $sprk-masthead-selector-dropdown-border-bottom-mask-open;
-  border-top: 0;
-  border-right: 0;
-  border-left: 0;
-  border-radius: 0;
+  border-radius: $sprk-border-radius;
   box-shadow: $sprk-masthead-selector-dropdown-box-shadow-mask-open;
+  font-size: $sprk-masthead-selector-dropdown-font-size;
+  position: absolute;
   max-width: 100%;
   margin: 0;
   width: 100%;
   top: 0;
   padding: $sprk-masthead-selector-dropdown-padding;
+  z-index: $sprk-layer-height-m;
 }
 
-.sprk-c-Masthead__selector-dropdown .sprk-c-Dropdown__link {
+.sprk-c-Masthead__selector-dropdown .sprk-c-Masthead__selector-dropdown-link {
   padding: $sprk-space-m $sprk-space-misc-a;
 }
 
+.sprk-c-Masthead__selector-dropdown-header {
+  padding: ($sprk-masthead-selector-dropdown-item-padding / 2)
+    $sprk-masthead-selector-dropdown-item-padding;
+
+  // GET RID? Might always have a heading
+  // When the header has a heading, this adjust its padding because the header takes over as the "first item"
+  + .sprk-c-Masthead__selector-dropdown-links {
+    .sprk-c-Dropdown__item:first-child
+      .sprk-c-Masthead__selector-dropdown-link {
+      padding-top: $sprk-masthead-selector-dropdown-item-padding / 2;
+    }
+  }
+}
+
+.sprk-c-Masthead__selector-dropdown-title {
+  color: $sprk-masthead-selector-dropdown-title-color;
+  font-weight: $sprk-masthead-selector-dropdown-title-font-weight;
+  font-size: $sprk-masthead-selector-dropdown-title-font-size;
+}
+
+.sprk-c-Masthead__selector-dropdown-links {
+  list-style-type: none;
+}
+
+.sprk-c-Masthead__selector-dropdown-item {
+  font-weight: $sprk-masthead-selector-dropdown-font-weight;
+
+  &:first-child .sprk-c-Masthead__selector-dropdown-link {
+    padding-top: $sprk-masthead-selector-dropdown-item-padding / 2;
+  }
+
+  &:last-child .sprk-c-Masthead__selector-dropdown-link {
+    padding-bottom: $sprk-masthead-selector-dropdown-item-padding / 2;
+  }
+}
+
+.sprk-c-Masthead__selector-dropdown-link {
+  color: $sprk-masthead-selector-dropdown-link-color;
+  display: block;
+  padding: ($sprk-masthead-selector-dropdown-item-padding / 2)
+    $sprk-masthead-selector-dropdown-item-padding;
+  text-decoration: none;
```

Comment:
We may want this to be a new $variable.

Reply:
Hey @RVMendoza, looks like you missed this comment from last review.